pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```diff
@@ -130,6 +130,8 @@ spec:
     workspaces:
     - name: output
       workspace: build-workspace-pv
+    - name: ssl-ca-directory
```

Comment:
We should also add this to the `git-clone-containerfile-repo` since it could be on the same git server with a custom cert

Reply:
👍